include/vrv/iomusxml.h

```diff
@@ -467,7 +467,7 @@ class MusicXmlInput : public Input {
     /* measure repeats */
     bool m_slash = false;
     /* MIDI ticks */
-    int m_ppq = 1;
+    int m_ppq = 0;
```

Comment:
can `m_ppq` ever be the denominator? If so, then this could lead to divide-by-zero.

Such as here? https://github.com/rism-digital/verovio/blob/6587876e2a7b336a85e2472c7d3bc54776ce2e9a/src/iomusxml.cpp#L528

Reply:
Maybe `-1` is better if you want to indicate that it's unset?

Reply:
Good point!